custom_components/tahoma/light.py

```diff
@@ -144,10 +144,13 @@ def update(self):
         states = self.tahoma_device.active_states
 
         self._state = states.get(CORE_ON_OFF_STATE) == STATE_ON
-        self._brightness = states.get(CORE_LIGHT_INTENSITY_STATE)
 
-        self._hs_color = color_util.color_RGB_to_hs(
-            states.get(CORE_RED_COLOR_INTENSITY_STATE),
-            states.get(CORE_GREEN_COLOR_INTENSITY_STATE),
-            states.get(CORE_BLUE_COLOR_INTENSITY_STATE),
-        )
+        if CORE_LIGHT_INTENSITY_STATE in states:
+            self._brightness = states.get(CORE_LIGHT_INTENSITY_STATE)
```

Comment:
What is the error you get? Because this should return None, which is the default value

cc @tetienne

Reply:
yes but it goes through `color_util.color_RGB_to_hs` which doesn't accept NoneType.

Reply:
Ok, makes sense, in that case you don't need this one for brightness, only for the hs_color.